src/codeu/chat/common/UserInterest.java

```diff
@@ -0,0 +1,48 @@
+package codeu.chat.common;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import codeu.chat.util.*;
```

Comment:
I'd generally avoid wildcard imports.

For more info, see [the official Java style guide](https://stackoverflow.com/documentation/java/2697/oracle-official-code-standard#t=201707101738451038974), [Google's style guide](https://google.github.io/styleguide/javaguide.html#s3.3-import-statements), and just for fun here's [Twitter's style guide](https://github.com/twitter/commons/blob/master/src/java/com/twitter/common/styleguide.md). :stuck_out_tongue: